docs/sources/whatsnew/whats-new-in-v10-1.md

```diff
@@ -0,0 +1,496 @@
+
+### Loki query splitting
+
+<!-- Matías Wenceslao Chomicki -->
+
+_Generally available in all editions of Grafana_
+
+In response to different query performance scenarios, we've implemented query splitting. Now, queries that request more than a day of data are split into sub-requests, each with a duration of one day. For example, requesting seven days of logs will produce seven requests of one day.
+
+### Easier to use query editor for Elasticsearch
+
+<!-- Gabor Farkas -->
+
+_Generally available in all editions of Grafana_
+
+The Elasticsearch query editor now allows convenient switching between logs, metrics, and raw data directly from the top, eliminating the need to go through the metric selector.
+
+### Metrics explorer
+
+The Metrics explorer is a new feature that enhances metrics browsing in the Prometheus query builder. The Metrics explorer makes it easier for you to find the right metric, and get comfortable with PromQL. You can now explore metrics with additional metadata, perform fuzzy search on the metric name or description, and filter on the Prometheus type.
+
+{{< figure src="/media/docs/grafana/screenshot-grafana-10-1-metrics-explorer.png" max-width="750px" caption="Searching in Metrics explorer" >}}
+
+## Redshift and Athena: Async query data support
+
+<!-- Isabella Siu, Kevin Yu, Andrés Martínez -->
+
+_Generally available in all editions of Grafana_
+
+Async query data support in Redshift and Athena makes queries over multiple requests (starting, checking status, and fetching the results) instead of in a single request query. This is useful for queries that can potentially run for a long time and time out. This feature was previously available behind a feature toggle and is now be generally available and enabled by default.
+
+## Redshift and Athena: Async query caching
+
+<!-- Isabella Siu -->
+
+_Experimental in Grafana Enterprise, Cloud Pro, and Cloud Advanced_
+
+This feature adds support for query caching of async queries in the Athena and Redshift data source plugins. To try this feature, enable both the `useCachingService` and `awsAsyncQueryCaching` feature toggles. If you’re using Grafana Cloud and would like to enable this experimental feature, please contact customer support.
+
+### CloudWatch logs Monaco query editor
+
+<!-- Isabella Siu, Kevin Yu -->
+
+_Experimental in all editions of Grafana_
+
+The CloudWatch logs query editor is moving from being a Slate-based editor to a Monaco-based editor. This new Monaco-based editor will provide improved syntax highlighting, and auto-completion. To use the Monaco-based query editor, enable the `cloudWatchLogsMonacoEditor` feature toggle. If you’re using Grafana Cloud and would like to enable this feature, please contact customer support.
+
+### InfluxDB Backend Mode
+
```

Comment:
@itsmylife - Should this section be under **Data sources** rather than a child of the the **Redshift and Athena: Async query caching** section?

**Note, please don't commit any changes without checking in with Quynton first.**

Reply:
@itsmylife - What is the availability of this feature? Experimental, private preview, public preview, or GA? And is it in available in Grafana OSS, Enterprise, Cloud or all three?

Also, I've made some edits to this entry. Please let me know if they're okay. 

**Note, please don't commit any changes without checking in with Quynton first.**

Reply:
1. Yes it should be under data sources. I thought I put it there. 
2. Both
3. They are perfect. Thanks.

Reply:
There was a nesting issue with the Redshift and Athena entries, so I just wanted to make sure of where this should live. Thanks!